packages/documentation/src/content/docs/integration/requirements/idp.mdx

```diff
@@ -12,179 +12,123 @@ import {
 
-The purpose of the `x-idp-secret` header is to secure communications between your IdP and authorization servers.
+## x-idp-secret header
 
-The header is used for requests to the following endpoints and its value should be a shared secret known to both entities.
+The `x-idp-secret` header is used for requests to the following endpoints:
 
 - `GET /grant/:id/:nonce`
 - `POST /grant/:id/:nonce/accept`
 - `POST /grant/:id/:nonce/reject`
 
-When your IdP server sends requests to your authorization server, your IdP must provide the secret via this header.
-
-To set up the header, set the [`IDENTITY_SERVER_SECRET`](#environment-variables) on your authorization server environment to a value that is also used to configure your IdP server's requests to your authorization server.
-
-### Sequence diagram
-
-The following diagram provides a high-level view of the flow from when a pending grant request is created through to the authorization server returning an access token to continue a successful (accepted) interaction.
-
-The diagram is for illustrative purposes and is not an exact representation of the flow. Additional information can be found in the <LinkOut href="https://openpayments.dev/introduction/grants/#identity-providers">Open Payments</LinkOut> documentation.
-
-<MermaidWrapper client:load hasBorder={false}>
-
-{/* prettier-ignore */}
-<Mermaid
-  graph={`sequenceDiagram
-   Client->>Authorization Server: Sends interactive outgoing payment grant request
-   Authorization Server-->>Client: HTTP 200 request successful 
-   Client->>Authorization Server: Starts user interaction session
-   Authorization Server->>Authorization Server: Sets session
-   Authorization Server-->>Client: HTTP 302 instructs client to redirect to Identity Provider
-   Client->>Identity Provider: Redirects end-user's browser to the Identity Provider's consent screen
-   Identity Provider->>Identity Provider: End-user accepts interaction
-   Identity Provider->>Authorization Server: Provides end-user's interaction choice
-   Authorization Server-->>Identity Provider: 202 HTTP choice accepted
-   Identity Provider->>Authorization Server: Instructs server to finish interaction
-   Authorization Server->>Authorization Server: Ends session
-   Authorization Server-->>Identity Provider: 302 HTTP instructs Identity Provider to redirect to client
-   Identity Provider->>Client: Redirects to Client
-   Client->>Client: Verifies hash
-   Client->>Authorization Server: Requests continuation of grant
-   Authorization Server->>Client: 200 HTTP OK, returns access token
-`}
-/>
-
-</MermaidWrapper>
+The header's purpose is to secure communications between your IdP and authorization server and its value should be a shared secret known to both entities. When your IdP server sends requests to your authorization server, your IdP must provide the secret via this header.
```

Comment:
Nit: 

> "When your IdP server sends requests to your authorization server" 

The `x-idp-secret` is specific to Rafiki's AS, and doesn't apply generally (ie if the ASE wants to run their own AS, they can add security in a way they see fit)